development/docker-compose-background.yml

```diff
@@ -17,7 +17,7 @@ version: '2'
 
 services:
   web-ui:
-    image: usdotfhwastol/carma-web-ui:latest
+    image: usdotfhwastol/carmawebui:latest
```

Comment:
I think this should still be `carma-web-ui`? That's what the build scripts for that repo should be outputting anyway.

Reply:
The build script outputs carmawebui as far as I could tell

Reply:
I'll have to take a look at that then, because that's not the expected result of that build script.

Reply:
How do you want to handle this. The same thing happens for the cohda driver